### PR TITLE
docs(cloud): add gRPC information to import-export article

### PR DESCRIPTION
**Pull request by bucanero**
Fixes DOC-203

 

Docs Preview: [Dgraph Preview](https://)

### CODE REVIEW

File: content/admin/import-export.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ You can export your data from one Dgraph Cloud backend, and then import this dat
 
 ## Exporting Data
 
-You can export your data using JSON format. To do this, call the `export` mutation on `/admin/slash`. As an example, if your GraphQL endpoint is at `https://frozen-mango.us-west-2.aws.cloud.dgraph.io/graphql`, then the /admin endpoint for the schema is be at `https://frozen-mango.us-west-2.aws.cloud.dgraph.io/admin/slash`.
+You can export your data using JSON format. To do this, call the `export` mutation on `/admin/slash`. As an example, if your GraphQL endpoint is at `https://frozen-mango.us-west-2.aws.cloud.dgraph.io/graphql`, then the `/admin` endpoint for the schema is at `https://frozen-mango.us-west-2.aws.cloud.dgraph.io/admin/slash`.
```
Comment (NamanJain8) [inline content/admin/import-export.md:12]: Not sure if we will rename `/admin/slash` to `admin/cloud`.

File: content/admin/import-export.md [hunk 2]
```diff
@@ -29,19 +29,28 @@ mutation {
...
+
+3. Run the live loader as follows:
+
+    ```
+    docker run -it --rm -v /path/to/g01.json.gz:/tmp/g01.json.gz dgraph/dgraph:v20.07-slash \
+      dgraph live --slash_grpc_endpoint=<grpc-endpoint>:443 -f /tmp/g01.json.gz -t <api-token>
+    ```
+
+{{% notice "note" %}}
+Running this via Docker requires you to use an unreleased tag (either `master` or `v20.07-slash`).
```
Comment (NamanJain8) [inline content/admin/import-export.md:55]: This note might need to be updated. This is using old tag.

Comment (bucanero) [reply content/admin/import-export.md:55]: > This note might need to be updated. This is using old tag.

which tag is the current one? `v20.11-slash`?

File: content/admin/import-export.md (hunk 1 above)
Comment (bucanero) [reply content/admin/import-export.md:12]: > Not sure if we will rename `/admin/slash` to `admin/cloud`.

if the current endpoint is `admin/slash` then we should keep it, until the change is public.

Comment (NamanJain8) [reply content/admin/import-export.md:12]: Ok.

File: content/admin/import-export.md (hunk 2 above)
Comment (NamanJain8) [reply content/admin/import-export.md:55]: I see `release/v20.11-slash` branch but somehow the tag was not added. I see `v20.07-slash` but not `v20.11-slash`. @antblood can you review this, please. I don't have the full context.

Comment (antblood) [reply content/admin/import-export.md:55]: yes, currently we're using `v20.11-slash`

**Review by antblood — APPROVED** [review]

File: content/admin/import-export.md (hunk 2 above)
Comment (bucanero) [inline content/admin/import-export.md:55]:
```suggestion
Running this via Docker requires you to use an unreleased tag (either `master` or `v20.11-slash`).
```

**Pull request merged**